src/attributes/components/AttributeListDatagrid/AttributeListDatagrid.tsx

```diff
@@ -120,6 +119,7 @@ export const AttributeListDatagrid = ({
             onToggle={handlers.onToggle}
           />
         )}
+        navigatorOpts={{ state: getPrevLocationState(location) }}
```

Comment:
if this is repeated across each view with the gridd, can just grid call `useLocation` inside of its component?